tgui/packages/tgui-panel/settings/middleware.js

```diff
@@ -33,22 +32,23 @@ export const settingsMiddleware = (store) => {
         store.dispatch(loadSettings(settings));
       });
     }
-    if (type === updateSettings.type || type === loadSettings.type) {
+    if (
+      type === updateSettings.type ||
+      type === loadSettings.type ||
+      type === addHighlightSetting.type ||
+      type === removeHighlightSetting.type ||
+      type === updateHighlightSetting.type
+    ) {
       // Set client theme
       const theme = payload?.theme;
       if (theme) {
         setClientTheme(theme);
-      } else if (type === loadSettings.type) {
-        updateSettings({
-          theme: 'dark',
-        });
```

Comment:
bad removal

Reply:
sorry I just kinda copy pasted it from the tg repo, and re-added our stuff back in